### ISSUE TITLE
Roles

### ISSUE THREAD

**Issue by mariuscoto** [rosedu/events#35]
Users can have different roles in an event.

Add a form to match a user to a role (from a predefined list of roles) in the context of an edition.

You can add or edit a role from the edition's page.
The roles are listed both on the edition page and on a users profile page.

I would go for:
* organiser - makes sure the event stays on track
* teacher - presents something to the participants
* mentor - work side by side with the participants
* participant

**Comment by mariuscoto** [comment]
@maria what do you think ? I would like to ask Alex to handle this issue, on CDL, this week.

**Comment by maria** [comment]
Maria:

- event creator this is a default, all events have creators. it's not necessary a role.
- event coordinator == organiser
- member of the organising team == teacher + mentor + anything else
- participant

I would go with:
   - lead organiser
   - organiser
   - mentor
   - mente

**Comment by mariuscoto** [comment]
I don't see why we should track the creator (inventor) of an event. The first two roles (lead organiser and organiser) and the next two (mentor and mentee) sound too much the same and it's hard to follow them. They seem to define the same category of people, so I would stick to this 'divisions' and to their initial names.

**Comment by maria** [comment]
> I don't see why we should track the creator (inventor) of an event.

I meant that now we show the person who created the event , and I think we should separate this person from the actual roles in the projects. For example I'm creating events even though I don't have a role in them.

**Comment by maria** [comment]
If we are speaking about events. I think we should add roles per events, not per editions. but we should listen to other opinions. @fullvlad @Aleks23 ?
